hackerank/js-practice/grading-students.js

```diff
@@ -0,0 +1,8 @@
+function gradingStudents(grades) {
+	return grades.map((studentGrade) => {
+		const round = Math.ceil((grades + 1) / 5) * 5;
+		if (studentGrade < 38) {
+			return studentGrade;
+		} else if (round - studentGrade < 3) return studentGrade + 5;
```

Comment:
When  `studentGrade` is greater than 38 and doesn't accomplish the `(round - studentGrade < 3)` doesn't return value. 

I think you need to add `return studentGrade` as the last step.